src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -122,11 +122,13 @@ public void ServicePoint_GetsCachedInstanceSpecificToHostPort()
         [Fact]
         public void ServicePoint_NetCoreApp_AddressIsAccessible()
```

Comment:
This test is poorly named and doesn't convey the real meaning of the test.  And now adding another item to the test just makes it worse.

It would be better to separate out the Assert.Throws() behavior to a new test method like "SendAsync_ServerDoesntExist_Throws()".   And to something like "ServicePoint_NetCoreApp_DefaultValues"